include/tscore/Diags.h

```diff
@@ -194,3 +194,17 @@ extern inkcoreapi Diags *diags;
 #define is_diags_on(_t) 0
 
 #endif // TS_USE_DIAGS
+
+#if !defined(TS_SVP)
+
+// Macros for convinient output of string_view (and thus also ts::TextView) instances using printf-like
+// functions.
+//
+#define TS_SVPF "%.*s"
```

Comment:
This functions kind of like the `PRIuMAX` and related types, right? Maybe call this TS_PRIsv?